cinema/views.py

```diff
@@ -1 +1,54 @@
-# write views here
+from rest_framework import mixins, viewsets
+
+from cinema.models import CinemaHall, Genre, Actor, Movie, MovieSession
+from cinema.serializers import GenreSerializer, CinemaHallSerializer, \
+    MovieSerializer, ActorSerializer, MovieSessionSerializer, \
+    MovieListSerializer, MovieDetailSerializer, MovieSessionListSerializer, \
+    MovieSessionDetailSerializer
+
+
+class GenreViewSet(mixins.ListModelMixin, mixins.CreateModelMixin,
```

Comment:
Maybe you want to remake your views.py with `ModelViewSet` ?)

![image](https://user-images.githubusercontent.com/104218550/179046904-0133cdce-73a1-4275-b5dc-b028613cfa3f.png)

Reply:
Of course, you're right, that's my bad.